---
source: tooling/nargo_cli/tests/execute.rs
expression: "&format!(\"success: {:?}\\nexit_code: {}\\n----- stdout -----\\n{}\\n----- stderr -----\\n{}\",\noutput.status.success(), output.status.code().unwrap_or(!0), stdout, stderr)"
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
error: 
  ┌─ src/main.nr:2:5
  │
2 │     std::static_assert(false, "");
  │     -----------------------------
  │
  = Call stack:
    1. src/main.nr:2:5

Aborting due to 1 previous error
